# Binomial Distribution
- The binomial distribution is a discrete probability distribution
- It describes the outcome of n independent trials in an experiment
- Each trial is assumed to have only two outcomes, either success or failure

### Example:
- Suppose there are twelve multiple choice questions
- Each question has five possible answers, and only one of them is correct
- Find the probability of having four or less correct answers if a student attempts to answer every question at random

#### Probability of answering a question correctly by random is 1/5=0.2

In [1]:
dbinom(4, size=12, prob=0.2)

[1] 0.1328756

- Probability of having 4 answers right is 13.3%

#### To find the probability of having four or less correct answers by random attempts, we apply the function dbinom with x = 0,…,4.

In [2]:
dbinom(0, size=12, prob=0.2) + 
dbinom(1, size=12, prob=0.2) + 
dbinom(2, size=12, prob=0.2) + 
dbinom(3, size=12, prob=0.2) + 
dbinom(4, size=12, prob=0.2) 

[1] 0.9274445

#### Same as finding lower tail probability using pbinom() function:

In [3]:
pbinom(4, size=12, prob=0.2, lower.tail=TRUE) 

[1] 0.9274445